auth0_flutter/README.md

```diff
@@ -38,6 +38,7 @@ Auth0 SDK for Android / iOS Flutter apps.
 
 - Web Auth login and logout
 - Automatic storage and renewal of user's credentials
+- Support for custom Credentials Manager implementations
```

Comment:
The abstract `CredentialsManager` was exported after all, as it is in the `auth0_flutter` package. So the respective documentation was added.